base_classes/NXtransformations.nxdl.xml

```diff
@@ -108,6 +126,143 @@
 		* depends_on
 
 		as needed.
+
+		**Example 1: goniometer**
+
+		Position a sample mounted on a goniometer in the :ref:`McSTAS coordinate frame&lt;Design-CoordinateSystem&gt;`.
+
+		The sample is oriented as follows
+
+		.. math:: X_\text{lab} = R(\vec{v}_\omega, \omega) . 
+								 T(\vec{v}_z, \text{sam}_z) . 
+								 T(\vec{v}_y, \text{sam}_y) . 
+								 T(\vec{v}_x, \text{sam}_x) . 
+								 R(\vec{v}_\chi, \chi) . 
+								 R(\vec{v}_\varphi, \varphi) . X_s
+
+		where
+
+		* :math:`R(\vec{v},a)` is a rotation around vector :math:`\vec{v}` with angle :math:`a`
+		* :math:`T(\vec{u},t)` is a translation along vector :math:`\vec{u}` over a distance :math:`t`
+		* :math:`X_s` a coordinate in the sample reference frame.
+
+		.. code-block::
+
+		  entry:NXentry
+		    sample:NXsample
+		      depends_on=transformations/phi
+		      transformations:NXtransformations
+		        phi=0
+		          @depends_on=chi
+		          @transformation_type=rotation
+		          @vector=[-1 -0.0037 -0.002]
+		          @units=degrees
+		        chi=0
+		          @depends_on=sam_x
+		          @transformation_type=rotation
+		          @vector=[0.0046 0.0372 0.9993]
+		          @units=degrees
+		        sam_x=0
+		          @depends_on=sam_y
+		          @transformation_type=translation
+		          @vector=[1 0 0]
+		          @units=mm
+		        sam_y=0
+		          @depends_on=sam_z
+		          @transformation_type=translation
+		          @vector=[0 1 0]
+		          @units=mm
+		        sam_z=0
+		          @depends_on=omega
+		          @transformation_type=translation
+		          @vector=[0 0 1]
+		          @units=mm
+		        omega=174
+		          @depends_on=.
+		          @transformation_type=rotation
+		          @vector=[-1 0 0]
+		          @units=degrees
+
+		**Example 2: different coordinate system**
+
+		Define a laboratory reference frame with the X-axis along the beam and the Z-axis opposite to the direction of gravity.
+		Three point detectors are positioned in this reference:
+
+		* *transmission*:
+			* point detector in the beam
+			* 20 cm downstream from the sample (the origin of the reference frame)
```

Comment:
the origin of WHICH reference frame (this means this lab frame we define, right?)